samples/authorization/csm_authorization_proxy_server_v200-alpha.yaml

```diff
@@ -42,11 +42,18 @@ spec:
           # enable: Enable/Disable csm-authorization proxy server
           enabled: true
           proxyService: dellemc/csm-authorization-proxy:v2.0.0-alpha
+          proxyServiceReplicas: 1
           tenantService: dellemc/csm-authorization-tenant:v2.0.0-alpha
+          tenantServiceReplicas: 1
           roleService: dellemc/csm-authorization-role:v2.0.0-alpha
+          roleServiceReplicas: 1
           storageService: dellemc/csm-authorization-storage:v2.0.0-alpha
+          storageServiceReplicas: 1
           opa: openpolicyagent/opa
           opaKubeMgmt: openpolicyagent/kube-mgmt:0.11
+          authorizationController: dellemc/csm-authorization-controller:v2.0.0-alpha
+          authorizationControllerReplicas: 1
+          leaderElection: false
```

Comment:
Leader elections should be enabled by default since the customer would encounter issues if they leave this as false but increase replicas of storage service.

Reply:
Made the changes. It has been set to true.